src/go/types/version/v2/schema.go

```diff
@@ -195,8 +196,7 @@ components:
           $ref: "#/components/schemas/Topology"
         scenario:
           nullable: true
```

Comment:
We are unable to launch any experiment that does not have a scenario file. Based on our knowledge, scheme v1 is used for launching experiments and not v2. We see in v2 you have `nullable: true` for scenarios. However, you do not have `nullable: true` in schema v1 for `scenario`. It is very possible this null bug is from an old branch and not this one specific.

*Error:*
`
creating experiment config: validating config after config hooks: config validation failed: Error at "/scenario": Value is not nullable Schema: { "properties": { "apps": { "items": { "properties": { "assetDir": { "example": "/phenix/topologies/example-topo/assets", "type": "string" }, "hosts": { "items": { "properties": { "hostname": { "example": "example-host", "type": "string" }, "metadata": { "additionalProperties": true, "example": { "setting0": true, "setting1": 42, "setting2": "universe key" }, "nullable": true, "type": "object" } }, "required": [ "hostname" ], "type": "object" }, "type": "array" }, "metadata": { "additionalProperties": true, "example": { "setting0": true, "setting1": 42, "setting2": "universe key" }, "nullable": true, "type": "object" }, "name": { "example": "example-app", "type": "string" } }, "required": [ "name" ], "type": "object" }, "type": "array" } }, "required": [ "apps" ], "type": "object" } Value: null
`